### PR TITLE
use named imports instead to avoid polluting the global namespace. in…

### PR DESCRIPTION
**Pull request by tony83033**
## What does this PR do?

in -> packages/emails/templates/_base-email.ts
The code imports nodemailer packages using default imports. It's recommended to use named imports instead to avoid polluting the global namespace.

## Type of change
- Chore (refactoring code, technical debt, workflow improvements)

### CODE REVIEW

File: packages/emails/templates/_base-email.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { decodeHTML } from "entities";
-import nodemailer from "nodemailer";
+import {createTransport} from "nodemailer";
```
Comment (Udit-takkar) [inline packages/emails/templates/_base-email.ts:2]: @tony83033  are there any benefits of importing only createTransport ?

File: yarn.lock
Comment (hariombalhara) [inline yarn.lock:1]: Unrelated change in yarn.lock. It needs to be reverted.

File: packages/emails/templates/_base-email.ts (hunk 1 above)
Comment (hariombalhara) [reply packages/emails/templates/_base-email.ts:2]: I think tree shaking works better this way.

**Review by hariombalhara — DISMISSED** [review]
There is an unrelated change in yarn.lock, otherwise it's good.

File: packages/emails/templates/_base-email.ts (hunk 1 above)
Comment (emrysal) [inline packages/emails/templates/_base-email.ts:2]:
```suggestion
import { createTransport } from "nodemailer";
```

**Review by emrysal — APPROVED** [review]

File: yarn.lock
Comment (tony83033) [reply yarn.lock:1]: Thanks, @hariombalhara  sir for pointing this out, actually I am new to the world of open source, could you please explain to me how can I revert it?

Comment (hariombalhara) [reply yarn.lock:1]: @emrysal took care of that.  I was talking about git revert and you can easily find articles on it 🙏

Comment (tony83033) [reply yarn.lock:1]: ohh got it Thanks
